src/views/video/VideoView.integrationTest.tsx

```diff
@@ -101,50 +101,83 @@ describe('Video View', () => {
     expect(await wrapper.findByLabelText('Copy video link')).toBeVisible();
   });
 
-  it('renders learning outcome if user has BO_WEB_APP_DEV flag', async () => {
-    fakeClient.users.setCurrentUserFeatures({ BO_WEB_APP_DEV: true });
-    fakeClient.videos.insertVideo(exampleVideo);
+  describe('Video AI Metadata', () => {
```

Comment:
I think it would be possible to also do testing at component level. The logic that we need to verify at video page level is whether the component is rendered depending on the feature flag setting.

But other tests like the contents themselves, loading states, error states can be tested at component level. Those tests will potentially require less setup and allow us to keep the files smaller.